src/main/java/seedu/duke/MainWindow.java

```diff
@@ -0,0 +1,51 @@
+package seedu.duke;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Duke duke;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/dragon.png"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/froggy.png"));
```

Comment:
I think these fields can be final since the images wont be changing.

Reply:
Thanks for the tip!